gandalf.js

```diff
@@ -5,62 +5,68 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.